modules/travel_pay/app/controllers/travel_pay/v0/claims_controller.rb

```diff
@@ -5,7 +5,8 @@ module V0
     class ClaimsController < ApplicationController
       def index
         begin
-          claims = service.get_claims(@current_user)
+          tokens = token_service.get_tokens(@current_user)
+          claims = claims_service.get_claims(tokens['veis_token'], tokens['btsss_token'])
```

Comment:
if you wanted to,  you could destructure the tokens hash:

```ruby
token_service.get_tokens(@current_user) => {'veis_token', 'btsss_token'}

# or if using symbols (see comment below)

token_service.get_tokens(@current_user) => {:veis_token, :btsss_token}
```

Reply:
Syntax is a little... goofy. Your call.

Reply:
Ah, there we go..... I was struggling with the whole syntax on that. I will say I was considering passing in the entire tokens object, then destructuring them in the `request_claims` call in the claims service.... 🤔